libtextsecure/message_receiver.js

```diff
@@ -900,7 +900,7 @@ MessageReceiver.prototype.extend({
   },
   async handleLokiAddressMessage(envelope, lokiAddressMessage) {
     const { p2pAddress, p2pPort } = lokiAddressMessage;
-    window.LokiP2pAPI.addContactP2pDetails(
+    window.lokiP2pAPI.addContactP2pDetails(
```

Comment:
Did you want to make this one using the global var directly like the ones above?